enterprise_gateway/itests/src/main.py

```diff
@@ -39,7 +40,8 @@ def init_nb_test_case(args):
                 if not target_kernels or nb_entity.kernel_spec_name in target_kernels:
                     nb_entities_list.append(nb_entity)
 
-    return NotebookTestCase(method, nb_entities_list, args.continue_when_error, args.host, args.username)
+    return NotebookTestCase(method, nb_entities_list, args.continue_when_error,
+                            args.host, args.username, args.impersonation_username)
```

Comment:
Let's just pass args, and let NotebookTestCase decide what it wants to pull out.  This way we don't modify the signature for each new argument.  Seems like we could use the `**kwargs` convention here.

Reply:
Good point. Will make the change.